src/component/canvas/LineRenderable.jsx

```diff
@@ -100,13 +107,22 @@ export class LineRenderable extends React.PureComponent {
     }
 
     render(): React.Element<any> {
+        const {
+            data,
+            line: Line,
+            xColumn,
+            xScale,
+            yColumn,
+            yScale
+        } = this.props;
+
         return <g>
-            <path
-                fill='none'
-                stroke='black'
-                strokeWidth='1'
-                {...this.props.pathProps}
-                d={this.buildPath()}
+            <Line
+                points={data.rows.map(row => [xScale(row.get(xColumn)), yScale(row.get(yColumn))]).toArray()}
```

Comment:
These points are upside down - y needs to be subtracted from the maximum of the range. This points array should probably be precomputed and passed to the buildPath function too, that way they are ensured to be the same and we avoid running the scales twice for each point.